redis_json/src/ivalue_manager.rs

```diff
@@ -290,6 +290,14 @@ impl<'a> WriteHolder<IValue, IValue> for IValueKeyHolderWrite<'a> {
         }
     }
 
+    fn notify_keyspace_event(&mut self, ctx: &Context, command: &str) -> Result<(), RedisError> {
```

Comment:
If we're splitting the functionality of `apply_changes` then we should have it call `notify_keyspace_event` to reduce duplication. eg something like
```rs
fn apply_changes(&mut self, ctx: &Context, command: &str) -> Result<(), RedisError> {
    self
        .notify_keyspace_event(ctx, command)
        .inspect(|_| ctx.replicate_verbatim())
}
```